content/to-other-languages/f-sharp-for-c-sharp-developers/f-sharp-overview-for-C-sharp.md

```diff
@@ -0,0 +1,159 @@
+---
+title: F# Overview for C# developers
+showMetadata: true
+editable: true
+showToc: true
+---
+
+This page intends to overview F# features to you. Once you finish this, you should be able to write simple F# code with functions :)
+
+# F# Introduction #
+
+ The key difference between F# and C# is that F# is a function-first language, meaning it expects us to write code full of functions and lambdas, and less of classes or even none.
+
+This article is not intended for introducing functional programming. I assume that the reader should have some knowledge about functional programming (i.e. its principles).
+
+Let's start from the high-level view...
+
+# Project and file structure #
+
+Here's the list of F# file extensions:
+
+| File Extension | Description     |
+| ----           | ----            |
```

Comment:
I am not sure this ender correctly but latest I suggest you to try this VS code extension which helps auto formatting Markdown table.

https://marketplace.visualstudio.com/items?itemName=fcrespo82.markdown-table-formatter